flink-connectors/flink-connector-pulsar/src/main/java/org/apache/flink/connector/pulsar/source/PulsarSourceBuilder.java

```diff
@@ -107,7 +107,7 @@
  *     .setSubscriptionName("flink-source-1")
  *     .setTopics(Arrays.asList(TOPIC1, TOPIC2))
  *     .setDeserializationSchema(PulsarDeserializationSchema.flinkSchema(new SimpleStringSchema()))
- *     .setUnboundedStopCursor(StopCursor.atEventTime(System.currentTimeMillis()))
+ *     .setUnboundedStopCursor(StopCursor.atPublishTime(System.currentTimeMillis()))
```

Comment:
I'm not sure this is correct, especially since the explanation above says `For example the following PulsarSource stops after it consumes up to a event time when the Flink started.`

Reply:
After I read the source code, I found the core logic of this [PR](https://github.com/apache/flink/pull/18345/files) is renaming `atEventTime` to `atPublishTime` and making `atEventTime` deprecated. @MartijnVisser 

@imaffe  What do you think?

Reply:
@imaffe What do you think?

Reply:
@syhily What do you think?

Reply:
This is ok.